lib/views/status.erb

```diff
@@ -154,9 +162,8 @@
                     </div>
                       <%= @remotes.first %>
                   </h6>
-                  <!-- TODO use git gem for status -->
                   <pre><%= @status %></pre>
-                  
+                  <%# TODO why is this here? %>
```

Comment:
[According to `git blame`](https://github.com/firstdraft/web_git/blame/5b952b270ef2bc91c89313b6fa4ae456efa37b8b/app/views/web_git/commands/status.html.erb#L118), it's from 3aeee54ff417a21a2e504c1b88428a45e227c583. Dunno how it connects to that based on the commit subject, but maybe we didn't want them pushing directly to `master` anymore?

Reply:
Yes, it was because the tooltip for `git push` should be different if the current branch is `master`/`main`. Other branches need the remote name and potentially the `-u` flag. The conditional can be condensed since only the tooltip changes.